experimental/torchprep/README.md

```diff
@@ -0,0 +1,127 @@
+# Torchprep
+
+A CLI tool to prepare your Pytorch models for efficient inference. The only prerequisite is a model trained and saved with `torch.save(model_name, model_path)`. See `example.py` for an example.
+
+**Be warned**: `torchprep` is an experimental tool so expect bugs, deprecations and limitations. That said if you like the project and would like to improve it please open up a Github issue!
+
+## Install from source
+
+Create a virtual environment 
+
+```sh
+apt-get install python3-venv
+python3 -m venv venv
+source venv/bin/activate
+```
+
+Install `poetry`
+
+```sh
+sudo python3 -m pip install -U pip
+sudo python3 -m pip install -U setuptools
+pip install poetry
+```
+
+Install `torchprep`
+
+```sh
+cd torchprep
+poetry install
+```
+
+## Install from Pypi (Coming soon)
+
+```sh
+pip install torchprep
+```
+
+## Usage
+
+```sh
+torchprep quantize --help
+```
+
+### Example
+
+```sh
+# Download resnet example
+python example.py
```

Comment:
@msaroufim seems transformers are missing from requirements, it has import error, and torchvision